pages/snake/index.js

```diff
@@ -0,0 +1,361 @@
+import ConfirmButton from "@/components/ConfirmButton/ConfirmButton";
+import Indicator from "@/components/Indicator/Indicator";
+import ArrowButtons from "@/components/SnakeGame/ArrowButtons/ArrowButtons";
+import Child from "@/components/SnakeGame/Child/Child";
+import Food from "@/components/SnakeGame/Food/Food";
+import Player from "@/components/SnakeGame/Player/Player";
+import StyledLink from "@/components/StyledLink/StyledLink";
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+
+const StyledSnakePage = styled.main`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+  width: 100vw;
+  height: 100vh;
+  background: #f1f3c2;
+  @media screen and (min-width: 600px) {
+    gap: 5vh;
+  }
+  @media screen and (min-width: 900px) {
+    gap: 10vh;
+  }
+`;
+
+const StyledGameField = styled.div`
+  display: grid;
+  place-content: center;
+  width: 300px;
+  min-height: 300px;
+  background: lightgreen;
+  border: 2px solid #000000;
+  border-radius: 20px;
+  position: relative;
+  border: 5px solid #000000;
+
+  @media screen and (min-width: 600px) {
+    transform: scale(1.2);
+  }
+  @media screen and (min-width: 900px) {
+    transform: scale(1.5);
+  }
+`;
+
+const StyledScoreContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 300px;
+  height: 100px;
+  font-size: 1.2rem;
+  font-weight: 800;
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-top: 40px;
+`;
+
+const StyledIndicatorContainer = styled.div`
```

Comment:
Please  avoid nesting in styled components, because it goes against the principle of modularity. Instead, create separate styled components for these elements. 🥹